core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -237,10 +237,8 @@ class RDDSuite extends SparkFunSuite with SharedSparkContext with Eventually {
 
   test("aggregate") {
     val pairs = sc.makeRDD(Seq(("a", 1), ("b", 2), ("a", 2), ("c", 5), ("a", 3)))
-    type StringMap = HashMap[String, Int]
-    val emptyMap = new StringMap {
-      override def default(key: String): Int = 0
-    }
+    type StringMap = scala.collection.mutable.Map[String, Int]
```

Comment:
`withDefaultValue` returns a inner class extending `mutable.Map`. So, we should use `Map` for `aggregate`.

Reply:
I think you can import mutable, and use `mutable.Map[String,Int]` instead of this type def, but, it's fine either way.
Looks like the only instance of this, so go ahead.